loafer/runners.py

```diff
@@ -48,12 +49,7 @@ def stop(self, *args, **kwargs):
             self._on_stop_callback()
 
         logger.info('cancel schedulled operations ...')
-        for task in asyncio.Task.all_tasks(self.loop):
-            task.cancel()
-            if task.cancelled() or task.done():
-                continue
-
-            with suppress(CancelledError):
-                self.loop.run_until_complete(task)
+        with suppress(CancelledError):
+            cancel_all_tasks(self.loop)
```

Comment:
Thanks for you collaboration @GuilhermeVBeira!
Do you have a sample case so I can reproduce?
I'm not sure if we should use a private function here, but maybe we can work on the expected behavior.

Reply:
> Do you have a sample case so I can reproduce?

Yes, I made another branch :
https://github.com/georgeyk/loafer/compare/master...GuilhermeVBeira:shutdow-example?expand=1
What happens some times you say to loop cancel the coroutines but he doesn't have a chance to do that, and you finish the execution leaking the CancelledError. This PR ensures you the loop cancels all currently coroutines before stopping the execution.

>I'm not sure if we should use a private function here

You are right, I updated it :man_facepalming: